### ISSUE TITLE
When unloading chemical combiner in some case makes it forget recipe and clog the whole machine

### ISSUE THREAD

**Issue by ESTPlayer** [al132mc/alchemistry#2]
The bug is as follows: 
1. Place chemical combiner
2. give it energy in any way
3. make recipe 4 carbon to 1 charcoal and lock it
4. make a system that supplies that carbon and pull charcoal out to somewhere else
5. Relog/unload chunks couple of times
6. Machine starts to overfill and clog up, recipe seems to be missing
7. Profit

Minecraft: 1.12.2
Forge:2681

Might but might not affect: Optifine, Foamfix, betterfps, vanillafix
Alchemistry version: 0.7.0
lib: 1.0.0

**Comment by al132mc** [comment]
I was able to duplicate this bug by simply setting a recipe, then relogging. Perhaps this packet issue is also what's causing #1 as well. Hopefully I can get this straightened out soon

**Comment by al132mc** [comment]
This should be fixed now in version 0.7.1 which I just released. Unfortunately I don't expect it to have fixed #1 as I had hoped..

**Comment by ESTPlayer** [comment]
This still happens in alchemistry 0.7.3, i had my base chunkloaded, went to mining world.. sent some items back using enderstorage bags, came back to base half a hour later, and the machine was completely clogged.

**Comment by al132mc** [comment]
Odd. I tried doing this presumably like you described (chunkloading machine with FTB utils in the overworld, setting a machine recipe, traveling to aroma's mining dimension, then sending back items to be piped in. When I went back to the overworld, even tried re-logging once or twice as well, the recipe was still set and working properly. I didn't spend 30 mins in the mining dimension, but i'm not sure why it would make a difference if it's chunloaded. I will keep looking, but this has me stumped for now...
